WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/EditableOrderViewModelTests.swift

```diff
@@ -1802,44 +1800,6 @@ final class EditableOrderViewModelTests: XCTestCase {
         XCTAssertEqual(orderItem.variationID, variationID)
         XCTAssertEqual(orderItem.quantity, 1)
     }
-
-    func test_order_with_no_products_when_updateOrderWithProduct_is_invoked_then_product_is_added() {
-        // Given
-        let product = Product.fake().copy(siteID: sampleSiteID, productID: sampleProductID, purchasable: true)
-        storageManager.insertSampleProduct(readOnlyProduct: product)
-        let viewModel = EditableOrderViewModel(siteID: sampleSiteID, storageManager: storageManager)
-
-        // Confidence check
-        let orderItem = viewModel.currentOrderItems.first
-        XCTAssertNil(orderItem)
-
-        // When
-        viewModel.updateOrderWithProductID(sampleProductID)
-
-        // Then
-        XCTAssertEqual(viewModel.currentOrderItems.count, 1)
-        XCTAssertEqual(viewModel.currentOrderItems.first?.quantity, 1)
-        XCTAssertEqual(viewModel.productRows[safe: 0]?.quantity, 1)
-    }
-
-    func test_order_with_products_when_updateOrderWithProduct_is_invoked_then_product_quantity_is_updated() {
-        // Given
-        let product = Product.fake().copy(siteID: sampleSiteID, productID: sampleProductID, purchasable: true)
-        storageManager.insertSampleProduct(readOnlyProduct: product)
-        let viewModel = EditableOrderViewModel(siteID: sampleSiteID, storageManager: storageManager, initialProductID: sampleProductID)
-
-        // Confidence check
-        let orderItem = viewModel.currentOrderItems.first
-        XCTAssertEqual(orderItem?.quantity, 1)
-
-        // When
-        viewModel.updateOrderWithProductID(sampleProductID)
-
-        // Then
-        XCTAssertEqual(viewModel.currentOrderItems.count, 1)
-        XCTAssertEqual(viewModel.currentOrderItems.first?.quantity, 2)
-        XCTAssertEqual(viewModel.productRows[safe: 0]?.quantity, 2)
-    }
```

Comment:
Should we keep these tests adapted to using a product model rather than a product ID?

Reply:
I removed these because `updateOrderWithProductID` is now private, and we test everything possible related to scanning below :)